pygmt/src/surface.py

```diff
@@ -42,7 +47,22 @@ def surface(data=None, x=None, y=None, z=None, **kwargs):
     .. math::    (1 - t)\nabla^2(z)+t\nabla(z) = 0
 
     where :math:`t` is a tension factor between 0 and 1, and :math:`\nabla`
-    indicates the Laplacian operator.
+    indicates the Laplacian operator. Here, :math:`t = 0` gives the
+    “minimum curvature” solution. Minimum curvature can cause undesired
+    oscillations and false local maxima or minima (See Smith and Wessel,
+    1990), and you may wish to use :math:`t > 0` to suppress these effects.
+    Experience suggests :math:`t \sim 0.25` usually looks good for potential
+    field data and t should be larger (:math:`t \sim 0.35`) for steep
+    topography data. :math:`t = 1` gives a harmonic surface (no maxima or
+    minima are possible except at control data points). It is recommended that
+    the user preprocess the data with :class:`pygmt.blockmean`,
+    :class:`pygmt.blockmedian`, or :class:`pygmt.blockmode` to avoid spatial
+    aliasing and eliminate redundant data. You may impose lower and/or upper
```

Comment:
```suggestion
    minima are possible except at control data points). It is recommended that
    the user preprocess the data with :func:`pygmt.blockmean`,
    :func:`pygmt.blockmedian`, or :func:`pygmt.blockmode` to avoid spatial
    aliasing and eliminate redundant data. You may impose lower and/or upper
```